CHANGELOG.md

```diff
@@ -1,11 +1,10 @@
-# NEXT RELEASE
+# 14.7.0 Release notes
 
 ### Enhancements
-* <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
 * Nested collections have full support for automatic client reset ([PR #7683](https://github.com/realm/realm-core/pull/7683)).
 
 ### Fixed
-* <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
+* Having links in a nested collections would leave the file inconsistent if the top object is removed. ([#7657](https://github.com/realm/realm-core/issues/7657), since 14.0.0)
```

Comment:
is this intentional? it seems it was released in 14.6.2

Reply:
This line was merged in the wrong section. You can verify by looking at the git history.